content-webapp/pom.xml

```diff
@@ -39,6 +39,12 @@
       <classifier>sources</classifier>
       <scope>provided</scope>
     </dependency>
+    <dependency>
```

Comment:
Are we sure that we do not create a circular dependency ?

Reply:
Nope, we don't, content depends on notes that's it,